tests/testthat/test-weekly.R

```diff
@@ -0,0 +1,643 @@
+# devtools::load_all()
+require(testthat)
+
+### everything needed for the tests in this file ###
+
+data("Flu2009")
+data("SARS2003")
+incid <- SARS2003$incidence
+dt <- 7L
+dt_vec <- c(2L,2L,3L)
+weekly_inc <- aggregate_inc(incid, dt)
+agg_inc <- aggregate_inc(incid, dt_vec)
+
+mean_si <- sum(SARS2003$si_distr * seq_along(SARS2003$si_distr))
+std_si <- sqrt(sum(SARS2003$si_distr * seq_along(SARS2003$si_distr)^2))
```

Comment:
incorrect, see https://socratic.org/questions/how-do-you-use-a-probability-mass-function-to-calculate-the-mean-and-variance-of